ocaml/xapi/import_raw_vdi.ml

```diff
@@ -112,17 +112,45 @@ let import vdi (req: Request.t) (s: Unix.file_descr) _ =
               debug "Checking whether localhost can see SR: %s" (Ref.string_of sr);
               if (Importexport.check_sr_availability ~__context sr)
               then localhost_handler rpc session_id vdi req s
-              else
-                let host = Importexport.find_host_for_sr ~__context sr in
-                let address = Db.Host.get_address ~__context ~self:host in
-                return_302_redirect req s address;
-                None
+              else 
+                begin
+                  try 
+                    let host = Importexport.find_host_for_sr ~__context sr in
+                    let address = Db.Host.get_address ~__context ~self:host in
+                    return_302_redirect req s address;
+                    None
+                  with 
+                  | Api_errors.Server_error (a,b) as e ->
+                    error "Caught exception in import handler: %s" (ExnHelper.string_of_exn e);
+                    let all = req.Request.cookie @ req.Request.query in 
+                    let task_id = 
+                    if List.mem_assoc "task_id" all 
```

Comment:
These three lines should be indented more. Can you run `ocp-indent` on the modified files before committing the changes please?